### PR TITLE
Add aliases to copy to and paste from clipboard on the terminal for linu...

### PR DESCRIPTION
**Pull request by mimoralea**
...x environments; analogous to pbpaste and pbcopy on Mac

### CODE REVIEW

File: aliases/available/clipboard.aliases.bash [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cite 'about-alias'
+about-alias 'pbcopy and pbpaste shortcuts to linux'
+
+case $OSTYPE in
+  linux*)
+    XCLIP=$(command -v xclip)
+    [[ $XCLIP ]] && alias pbcopy="$XCLIP -selection clipboard" && alias pbpaste="$XCLIP -selection clipboard -o"
+    ;;   
+  darwin*)
```
Comment (nwinkler) [inline aliases/available/clipboard.aliases.bash]: Do we need the case for Darwin? It's not doing anything - can't we remove it?

Comment (mimoralea) [inline aliases/available/clipboard.aliases.bash]: We sure can! Will do in a sec.

**Comment by mimoralea** [comment]
I wonder why it failed, can't see the issue being related to removing those two lines.

**Comment by nwinkler** [comment]
The build failure is unrelated, don't worry about it.

**Comment by mimoralea** [comment]
Cool

**Pull request merged**